cypress/e2e/DetailsPage_spec.cy.js

```diff
@@ -119,11 +119,18 @@ describe("Details Page (bad response)", () => {
     cy.wait("@gqlshowDetailsQuery")
   })
 
-  it("should not show a table row for streaming providers if there are none available", () => {
+  it("should redirect to Error Component if there is a bad response", () => {
     cy.url().should("eq", "http://localhost:3000/error")
+    cy.get('.error').should('be.visible')
+    cy.get('[d="M13.768 4.2C13.42 3.545 12.742 3.138 12 3.138s-1.42.407-1.768 1.063L2.894 18.064a1.986 1.986 0 0 0 .054 1.968A1.984 1.984 0 0 0 4.661 21h14.678c.708 0 1.349-.362 1.714-.968a1.989 1.989 0 0 0 .054-1.968L13.768 4.2zM4.661 19 12 5.137 19.344 19H4.661z"]').should('be.visible')
+    cy.get('[d="M20 3H4a2 2 0 0 0-2 2v4a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V5a2 2 0 0 0-2-2zM4 9V5h16v4zm16 4H4a2 2 0 0 0-2 2v4a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2v-4a2 2 0 0 0-2-2zM4 19v-4h16v4z"]').should('be.visible')
```

Comment:
This definitely works, but if you're trying to shorten some of these really long `.get`s in the future, you can also assign a cypress-specific attribute to the element like `data-cy="element-description"`. It also helps if you refactor class names (or icons from that library, in this case) in the future, by ensuring that you continue to target the element and your tests don't break. [The docs go into detail on it here.](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) 

I know you said you were trying to pick up some new Cypress skills, so thought I'd share!